app/views/application/equality-monitoring/ethnic-group.html

```diff
@@ -9,38 +9,49 @@
 {% endblock %}
 
 {% block primary %}
+
+  <p class="govuk-body">Training providers will not see your answer to this question when deciding whether to offer you a place.</p>
+
+  <p class="govuk-body">They will only see your answer if you accept an offer from them.</p>
+
   {{ govukRadios({
+    fieldset: {
+      legend: {
+        text: "What is your ethnic group?",
+        classes: "govuk-fieldset__legend--m"
+      }
+    },
     items: [{
-      text: "Asian or Asian British",
+      text: "White",
       label: {
         classes: "govuk-label--s"
       },
       hint: {
-        text: "Includes any Asian background, for example, Bangladeshi, Chinese, Indian, Pakistani"
+        text: "Includes any White background"
       }
-    }, {
-      text: "Black, African, Black British or Caribbean",
+    },{
+      text: "Mixed or multiple ethnic groups",
       label: {
         classes: "govuk-label--s"
       },
       hint: {
-        text: "Includes any Black background"
+        text: "Includes any Mixed background"
       }
-    }, {
-      text: "Mixed or multiple ethnic groups",
+    },{
+      text: "Asian or Asian British",
       label: {
         classes: "govuk-label--s"
       },
       hint: {
-        text: "Includes any Mixed background"
+        text: "Includes any Asian background, for example, Bangladeshi, Chinese, Indian, Pakistani"
       }
     }, {
-      text: "White",
+      text: "Black, African, Caribbean or Black British",
```

Comment:
The Census used `Black, Black British, Caribbean or African` for this option.

Reply:
I took the wording from the [Equality information pattern](https://design-system.service.gov.uk/patterns/equality-information/). Not sure if the ordering is deliberately different from the 2021 (?) census or not.

Reply:
Will stick with the GDS pattern for now, and raise this question upstream.